### DISCUSSION TITLE
Make the address bar background dark on ios devices

### DISCUSSION THREAD

**Discussion by dbzx10299** [mdn/mdn-community#506] [Platform] [answered]
On ios the address bar is a bright white color which makes reading the docs very annoying at night on a mobile device. On chrome it is dark, on ios it would be great if it could also be dark. 

![78BFE4FD-12C1-4B3E-866B-EBCE1EDC9531](https://github.com/mdn/content/assets/40828283/3681ef22-11d7-4b39-90c3-c83d2ce758f4)

**Comment by bsmth** [comment, marked as answer]
I think this is a duplicate of https://github.com/orgs/mdn/discussions/156 so I would close for now in favor of the other one. Do you agree?

Thanks a lot!
